dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/serial/SerializingExecutor.java

```diff
@@ -42,7 +42,7 @@ public final class SerializingExecutor implements Executor, Runnable {
      */
     private final AtomicBoolean atomicBoolean = new AtomicBoolean();
 
-    private final ExecutorService executor;
```

Comment:
Why change type from ExecutorService to Executor?
I think there's no need to change it.

Reply:
Already rolled back.